File: 01.ProgrammingFundamentals/Java/01.IntroAndBasicSyntax/Exercises/src/P03_MilesToKm.java
import java.util.Scanner;
public class P03_MilesToKm {
    public static void main(String[] args) {

        Scanner console = new Scanner(System.in);
        double miles = Double.parseDouble(console.nextLine());
        System.out.format("%.2f",miles * 1.60934);
    }
}

File: ProgrammingFundamentals/C#/01.IntroAndBasicSyntax/Exercises/P03_MilesToKm/P03_MilesToKm.cs
using System;

namespace P03_MilesToKm
{
    class P03_MilesToKm
    {
        static void Main(string[] args)
        {
            double miles = double.Parse(Console.ReadLine());
            Console.WriteLine(String.Format("{0:0.00}", miles* 1.60934));
        }
    }
}
